base_agent/documents/logical_form_specification/conditions.md

```diff
@@ -98,3 +98,13 @@ TIME_CONDITION = {
 ```
 Note that `event` in time_condition represents when to start the timer and
 `input_left` by default in time condition marks the time since the event condition.
+
+
+## RemoveCondition ##
+
+"remove_condition": {"condition": {"comparison_type": "EQUAL",
```

Comment:
is this meant to be temporary? I guess we will reorganize all the condition/event docs after the later PRs get put in?